.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I'd rather not include this, we decided awhile back to exclude VSCode configuration files.